test/hexdocs/queue_test.exs

```diff
@@ -2,12 +2,13 @@ defmodule Hexdocs.QueueTest do
   use ExUnit.Case, async: true
   import Hexdocs.TestHelper
   alias Hexdocs.{HexpmMock, Store}
-  alias Hexdocs.Queue.Consumer
 
   @bucket :docs_private_bucket
   @public_bucket :docs_public_bucket
 
   setup do
+    Mox.set_mox_global()
```

Comment:
this is needed because now we're testing it through the pipeline so the consumer code is executed in a separate process.